dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7716,16 +7662,8 @@ public static final class Name {
         "alluxio.master.network.keepalive.timeout";
     public static final String MASTER_NETWORK_PERMIT_KEEPALIVE_TIME_MS =
         "alluxio.master.network.permit.keepalive.time";
-    public static final String MASTER_PERSISTENCE_INITIAL_INTERVAL_MS =
-        "alluxio.master.persistence.initial.interval";
     public static final String MASTER_PERSISTENCE_MAX_TOTAL_WAIT_TIME_MS =
```

Comment:
This was referred by `USER_FILE_PERSISTENCE_INITIAL_WAIT_TIME`. Instead of having to remove `USER_FILE_PERSISTENCE_INITIAL_WAIT_TIME` and all the dependencies, I chose to keep it around for a while. The user-side property will be removed soon and this one will be gone with that in the next batch of cleanup. This compromise helps keeping this PR smaller.